packages/relic/lib/src/body/types/mime_type.dart

```diff
@@ -0,0 +1,61 @@
+import 'dart:io';
+
+class MimeType {
+  static const plainText = MimeType('text', 'plain');
+
+  static const html = MimeType('text', 'html');
+
+  static const css = MimeType('text', 'css');
+
+  static const csv = MimeType('text', 'csv');
+
+  static const javaScript = MimeType('text', 'javascript');
+
+  static const json = MimeType('application', 'json');
+
+  static const xml = MimeType('application', 'xml');
+
+  // Binary
+
+  static const binary = MimeType('application', 'octet-stream');
+
+  static const pdf = MimeType('application', 'pdf');
+
+  static const rtf = MimeType('application', 'rtf');
+
+  final String primaryType;
+  final String subType;
+
+  const MimeType(this.primaryType, this.subType);
+
+  factory MimeType.parse(String type) {
```

Comment:
Suggestion: This will be hard to use without inspecting the contents of the method. Please add a description to these methods.